javalib/src/main/scala/java/util/Collections.scala

```diff
@@ -0,0 +1,1131 @@
+package java.util
+
+import java.{lang => jl, io}
+
+import scala.annotation.tailrec
+import scala.language.implicitConversions
+import scala.collection.JavaConversions._
```

Comment:
Language imports should come first, then annotation imports, then the rest, with blank lines in between.